.gitignore

```diff
@@ -129,4 +129,11 @@ geckodriver.exe
 geckodriver.log
 
 #Chrome Driver
-chromedriver.exe
\ No newline at end of file
+chromedriver.exe
+
+#Terraform_files
+.terraform.lock.hcl
```

Comment:
The lock file should actually be committed. It's needed to keep versions of plugins correct... similar to the role of poetry.lock.

Reply:
Removed from here and Dockerignore

Reply:
(In the upcoming Pull request)